app/controllers/wishlists_controller.rb

```diff
@@ -69,4 +61,11 @@ def update
     def wishlist_params
         params.require(:wishlist).permit(:title, :description, :color, :private, :organization_id)
     end
+
+    def set_organization
+      subdomain = request.subdomain
+      @organization = Organization.find_by(subdomain:)
+
+      redirect_to root_url(subdomain: 'www'), alert: 'Organization not found' unless @organization
```

Comment:
<img width="1512" alt="Screenshot 2023-09-19 at 22 24 04" src="https://github.com/wyshlist/wyshlist/assets/74747669/946faf1e-2b02-43e8-a8fa-fc4a182235ee">

this throws an error, I've seen this before, I think you need to add `redirect_to root_url(subdomain: 'www'), allow_other_host: true`